roles/kubespray-defaults/defaults/main/checksums.yml

```diff
@@ -1141,6 +1141,7 @@ nerdctl_archive_checksums:
     1.0.0: 2fb02e629a4be16b194bbfc64819132a72ede1f52596bd8e1ec2beaf7c28c117
 containerd_archive_checksums:
   arm:
+    1.7.12: 0
```

Comment:
@KubeKyrie  you must have missed `runc_checksums`. Please update it.

https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubespray-defaults/defaults/main/checksums.yml#L703

Reply:
done, thanks